__tests__/app.test.js

```diff
@@ -60,8 +63,42 @@ describe("/api/articles/:article_id", () => {
     return request(app)
       .get("/api/articles/invalid-id")
       .expect(400)
-      .then(({body}) => {
-        expect(body.msg).toBe("Bad request")
-      })
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+});
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET: 200 responds with an array of comments for the given article id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length).toBe(11);
+        body.forEach((comment) => {
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.article_id).toBe("number");
+          expect(typeof comment.created_at).toBe("string");
+        });
+      });
+  });
+  test("GET: 404 sends an appropriate error status alongside an error message when given a valid but non-existent id", () => {
+    return request(app)
+      .get("/api/articles/1000/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Article does not exist");
+      });
+  });
+  test("GET: 400 sends an appropriate error status alongside an error message when given an invalid id", () => {
+    return request(app)
+      .get("/api/articles/invalid-id/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
```

Comment:
Please add a test where it responds with an empty array of comments and status 200.